app.py

```diff
@@ -57,7 +57,7 @@ def node_register():
         node = Node(node_key)
 
     node.ip = request.remote_addr
-    node.connection_config = request.data
+    node.proposals = json.dumps([proposal])
```

Comment:
row should have 1 poposal, but not list of proposals.
`proposals` -> `proposal`

Reply:
row.proposals -> json.proposals mapping should happen in API controller layer

Reply:
Done.